src/host/premake.c

```diff
@@ -365,7 +365,7 @@ int premake_locate_executable(lua_State* L, const char* argv0)
 	}
 #endif
 
-#if PLATFORM_BSD
+#if PLATFORM_BSD && !defined(__OpenBSD__)
```

Comment:
This seems not great - is there *something* that can be done to support OpenBSD in this block instead? What does this block do on OpenBSD? I got it working on FreeBSD by adding that `sysctl` call, is there something similar for OpenBSD?

Sorry to bombard you with questions! I'm keen to see what we can do to improve the BSD situation!

Reply:
That's fair enough, perhaps wrapping that block instead would be better? Not required, if the none of it will work on OpenBSD it's probably pointless reducing it down to what doesn't compile.

Reply:
I meant that entire `if` block that contains the call to `sysctl`, but it's not really worth the effort. Thanks for the link, it's a pretty interesting discussion.